spring-cloud-vault-core/src/main/java/org/springframework/cloud/vault/VaultResponse.java

```diff
@@ -16,11 +16,11 @@
 
 package org.springframework.cloud.vault;
 
-import java.util.Map;
+import lombok.Data;
```

Comment:
Thanks but that's what I imported.  Import ordering seems to have a separate import/export from formatting (Java->Code Style->Organize Imports in Eclipse).  Will keep a closer eye on import changes moving forwards